Header header
int8 id
string name
bool is_ready
bool is_arrived
float32 pose_x
float32 pose_y
